workflows/nanoseq.nf

```diff
@@ -64,7 +64,11 @@ if (!params.skip_demultiplexing) {
 
     if (params.barcode_kit && qcatBarcodeKitList.contains(params.barcode_kit)) {
         if (params.input_path) {
-            ch_input_path = Channel.fromPath(params.input_path, checkIfExists: true)
+            if (workflow.profile.contains('test')){
+                ch_input_path = params.input_path
+            } else {
+                ch_input_path = Channel.fromPath(params.input_path, checkIfExists: true)
+            }
```

Comment:
I don't really understand this, there is no difference in the way the channel gets taken right?

Reply:
These are different.

For the test, I need to get the stage fast5 directory (with many fast5 files) from nf-core/test-dataset, so the input_path is not local, so checkInExist doesn't work

For the user input, there's no staging of the fast5 directory, so checking whether those exist is needed.

Reply:
Ah sorry, misread `input_path` as just `input` :sweat_smile:  . I find the testdata set up unusual which is also why tripped me up, but I don't think this is relevant for this PR (also given you've been waiting such a long time)